sttp/build.sbt

```diff
@@ -14,4 +15,8 @@ val `sttp-client` =
         "com.typesafe.akka" %% "akka-stream" % "2.5.18" % Test
       )
     )
-    .dependsOn(`algebra-jvm` % "compile->compile;test->test", `algebra-playjson-jvm` % "test->test")
+    .dependsOn(
+      `algebra-jvm` % "compile->compile;test->test",
+      `algebra-playjson-jvm` % "test->test",
+      `json-schema-circe-jvm`
+    )
```

Comment:
Should we introduce a `sttp-client-circe` module instead of adding a dependency to the `json-schema-circe` module here?